api/src/db/pipelines.rs

```diff
@@ -34,10 +47,13 @@ pub async fn create_pipeline(
     source_id: i64,
     destination_id: i64,
     image_id: i64,
-    publication_name: &str,
-    config: &PipelineConfig,
-) -> Result<i64, sqlx::Error> {
+    config: PipelineConfig,
+) -> Result<i64, PipelinesDbError> {
+    // TODO: remove publication name since it's in the config.
```

Comment:
Forgot to handle this todo?

Reply:
This is kept since we need the publication name column to be removed from the DB before removing this all over the place.